### ISSUE TITLE
click Generate Suggestions button received html source code, do I need other libraries to render it?

### ISSUE THREAD

**Issue by pkumc** [tobegit3hub/advisor#36]
![image](https://user-images.githubusercontent.com/9345057/56113021-383d1500-5f8f-11e9-85eb-18eea76741da.png)

**Comment by tobegit3hub** [comment]
Thanks for reporting. You don't need to install anything. 

This maybe a bug of response data and how can you re-produce this issue?

**Comment by Lady-Ariel** [comment]
yes,and i run examples/simple_function_example/min_function.py trial = client.get_suggestions(study.id, 1)[0] the trail returns '[]',
